packages/venia-ui/lib/components/MiniCart/ProductList/item.js

```diff
@@ -20,38 +21,44 @@ const Item = props => {
         quantity,
         configurable_options,
         handleRemoveItem,
-        prices
+        prices,
+        closeMiniCart
     } = props;
+
     const classes = mergeClasses(defaultClasses, propClasses);
+    const itemLink = useMemo(
+        () => resourceUrl(`/${product.url_key}${product.url_suffix}`),
+        [product.url_key, product.url_suffix]
+    );
 
-    const { isDeleting, removeItem } = useItem({ id, handleRemoveItem });
+    const { isDeleting, removeItem } = useItem({
+        id,
+        handleRemoveItem
+    });
 
     const rootClass = isDeleting ? classes.root_disabled : classes.root;
 
-    const deleteButton = !isDeleting ? (
-        <button
-            className={classes.editButton}
-            onClick={removeItem}
-            type="button"
-        >
-            <Icon
-                size={16}
-                src={DeleteIcon}
-                classes={{ icon: classes.editIcon }}
-            />
-        </button>
-    ) : null;
-
     return (
         <div className={rootClass}>
-            <Image
-                alt={product.name}
-                classes={{ root: classes.thumbnail }}
-                width={100}
-                resource={product.thumbnail.url}
-            />
-            <span className={classes.name}>{product.name}</span>
-            {deleteButton}
+            <Link
+                className={classes.thumbnail}
+                to={itemLink}
+                onClick={closeMiniCart}
+            >
+                <Image
+                    alt={product.name}
+                    classes={{ root: classes.thumbnail }}
+                    width={100}
+                    resource={product.thumbnail.url}
+                />
```

Comment:
For some reason the border around the image got thicker in this PR vs develop:

[![Image from Gyazo](https://i.gyazo.com/72842e45f2b3c2327860eecf18420d61.png)](https://gyazo.com/72842e45f2b3c2327860eecf18420d61)